specs/Standards/MultiToken/README.md

```diff
@@ -0,0 +1,538 @@
+- Proposal Name: Multi-Token-Standard
+- Start Date: 2021/07/24
+- Issue(s): #245.
+# Summary
+[summary]: #summary
+
+A standard interface for  a multi token standard that supports fungible, semi-fungible, and tokens of any type, allowing for ownership, transfer, and batch transfer of tokens generally regardless of specific type.
+
+# Motivation
+[motivation]: #motivation
+Having a single contract represent both NFTs and FTs can greatly improve efficiency as demonstrated by Enjin Coin. The ability to make batch requests with multiple asset classes can reduce a many transactions transaciton to a single transaction to trade around both NFTs and FTs that are a part of same token contract.
+
+Having this will also increase Near's ability to work interoperably with other chains. This will reduce the complexity required to represent these emerging asset classes.
+
+
+Prior art:
+- EIP-1155 : https://github.com/ethereum/EIPs/issues/1155
+
+- This NEP derives some examples and format from: https://github.com/near/NEPs/pull/21
+
+- Example of an NFT Series: https://github.com/near-apps/gnr8
+
+- Things from the NFT Discussions: https://github.com/near/NEPs/discussions/171
+
+- Things from the NFT Discussions: https://gov.near.org/t/nft-standard-discussion/853
+
+Discussions out of band:
+ - https://gov.near.org/t/multi-token-standard-discussion/2917
+ - https://github.com/shipsgold/multi-token-standard-impl/tree/main/meetings 
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+We should be able to do the following:
+- Get balance of token per user
+- Get balance in batch of a user
+- Get supply of tokens per id
+- Get supply of tokens in batch per id
+- Represent non fungibility of tokens
+- Represent fungibility of tokens
+- Transfer tokens by id in batch
+- Use these tokens on an exchange
+- Refund storage costs for fungible tokens
+
+There are a few concepts in the scenarios above:
+- **Total supply**. It's the total number of tokens in circulation.
+- **Balance owner**. An account ID that owns some amount of tokens.
+- **Transfer**. Moves some amount from one account to another account.
+- **Fungibility**. An indistinguishable amount of tokens to exchange 
+- **Non Fungibility**. Tokens that are differentiable from each other.
+
+### Real scenarios
+ 
+#### Simple transfer
+
+Alice wants to send 5 gold tokens to Bob.
+
+Let's assume the following:
+- The `gold` token is represented by the `games` with token_id `g133`.
+- Alice's account is `alice`.
+- Bob's account is `bob`.
+- The precision for `gold` on the games contract is `10^8`. 
+- The 5 tokens is `5 * 10^8` or as a number is `500000000`. 
+
+High-level explanation:
+
+Alice needs to issue one transaction to `games` contract to transfer 5 tokens (multiplied by precision) to Bob.
+ 
+Technical calls:
+
+1. `alice` calls `games::mt_transfer({"receiver_id": "bob", "amount": "500000000", "token_id": "g133"})`.
```

Comment:
Even for a simple transfer, it would be better to let user choose to have a msg payload (with max 64 bytes long) as remark or memo to this transfer. as this way:
```suggestion
 `games::mt_transfer({"receiver_id": "bob", "amount": "500000000", "token_id": "g133", "memo": "for my dinner"})`.
```